sphinx_gallery/_static/sg_gallery.css

```diff
@@ -74,23 +74,26 @@ thumbnail with its default link Background color */
 
 .sphx-glr-script-out {
   color: #888;
-  margin: 0;
+  display: flex;
+  gap: 0.5em;
 }
-p.sphx-glr-script-out {
-    padding-top: 0.7em;
+.sphx-glr-script-out::before {
+  content: "Out:";
+  line-height: 1.4;
+  padding-top: 10px;
 }
 .sphx-glr-script-out .highlight {
   background-color: transparent;
-  margin-left: 2.5em;
-  margin-top: -2.9em;
+  flex-grow: 1;
+  overflow: auto;
 }
 .sphx-glr-script-out .highlight pre {
   background-color: #fafae2;
   border: 0;
   max-height: 30em;
   overflow: auto;
   padding-left: 1ex;
-  margin: 0px;
+  margin: 2px;
```

Comment:
This is necessary in the pydata sphinx theme because `pre` has a box shadow which would be clipped by the `overflow:auto` in the parent `div` above..